mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go

```diff
@@ -408,7 +414,7 @@ func (p EBSPlugin) FetchMetrics() (map[string]interface{}, error) {
 			for _, metric := range graphdef[graphName].Metrics {
 				metricKey := graphName + "." + metric.Name
 				cloudwatchdef := cloudwatchdefs[metricKey]
-				val, err := p.fetch(vol, cloudwatchdef)
+				val, err := p.fetch(context.TODO(), vol, cloudwatchdef)
```

Comment:
[memo]FetchMetrics func is a required interface by go-mackerel-plugin-helper. This can't pass Context now.
https://github.com/mackerelio/go-mackerel-plugin-helper/blob/61befdeede9fbb889f1f367cc1785fb3b52fba7f/mackerel-plugin.go#L50